formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/FormstackConfig.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.identity.formstackbatonrequests
+
+case class SarLambdaConfig (
+  resultsBucket: String,
+  resultsPath: String,
+  performSarFunctionName: String
+)
+
+object FormstackConfig {
+
+  private def getEnvironmentVariable(env: String): Option[String] =
```

Comment:
I moved this into a function mostly because it's used a number of times (and when I add the second lambda to this project, it'll be called even more). It's not saving me writing that much code but I thought maybe it made it more explicit what wrapping this in an Option and System.getenv are doing